internal/manifest/manage.go

```diff
@@ -25,3 +28,48 @@ func Create(path string) error {
 	}
 	return filesystem.WriteFile(path, formattedInitManifest)
 }
+
+// Loads, validates, and unmarshals the manifest file at the given path into a flagset
+func Load(manifestPath string) (*flagset.Flagset, error) {
```

Comment:
good call, I think moving this logic here makes sense 👍

Reply:
that's a good point, the problem I had keeping it in the flagset file was a circular dependency occurring where I needed the type defined in flagset in this package, but the flagset file also imported the manifest package

I wouldn't mind renaming this to be more specific to loading the flagset though!